_sass/jekyll-theme-minimal.scss

```diff
@@ -107,7 +107,7 @@ img {
 header {
   width:270px;
   float:left;
-  position:fixed;
+  //position:fixed;
```

Comment:
Commenting code is not a way of removing code. I removed this comment in https://github.com/ta4j/ta4j-wiki/pull/11